lms/static/sass/mobile/_main.scss

```diff
@@ -0,0 +1,6 @@
+body {
+  @include padding-left($baseline);
+  @include padding-right($baseline);
```

Comment:
Nit: this is fine as is, but in the future, when the right and left sides are the same you don't need to use include. the include mixin is used for i18n. So this could be padding: 0 $baseline;

Reply:
@HarryRein Go it, thanks!